wiki/MetaWiki/en.md

```diff
@@ -0,0 +1,128 @@
+
+[GitHub supports GPG signature verification as written in their blog post][GH Blog GPG].
+
+It is a personal proof that the commit made by the account user personally, not from a malicious account using the same name.
+By default, all accounts will **not** have the "Verified" button, as it is an optional commit integrity feature.
+
+To get the "Verified" button, it will take a while; from downloading a tool, generation of the key from the tool, and some careful configuration.
+
+- For a Windows-only guide, check [the mrexodia's blog post guide by Duncan][GPG guide by Duncan]
+- For a GitHub's guide, check the [GitHub index page for GPG][GH Help GPG]
+
+Do note that it is only applied to **local Git usage only**.
+
+## Help!
+
+Use the search engine of your choice and type in your help request.
+If you cannot find the answer, you are not trying hard enough or your question is too ambiguous.
+Use only the keywords or refine your search and try again.
+Usually, copy-and-paste the issue code directly might get you an answer, but effectiveness varies.
+
+For further help, you can ask someone in the [osu!dev Discord chat][osu!dev Discord] (``#osu-wiki`` channel).
+Make a Discord account if you have not and go to the `#arrival` channel first if this is your first time to state your reason of being here.
+Depending on severity of the problem and members' active time, you may or may not get an answer.
+
+You _could_ also make an issue in [osu-wiki's Issues tab][GH Issue] to get help; however, the response time will vary and likely to be ignored if it is off-topic or is a trivial issue.
+
+## What happened to the old osu!wiki?
+
+Deprecated due to known performance issues and no solid solution to prevent spam.
+
+## How would this differ from the old wiki (running on MediaWiki)?
+
+osu!wiki will be ran like a git repo (repository).
+You can make mass changes easily and efficiently, and you can delay your changes for as long as you like.
+
+Unlike the old osu!wiki, changes has to be made via pull requests which are then reviewed for errors and myths before merging by someone with write permission to the master branch.
+This adds a layer of security against spammers and vandals.
+
+Addition, deletion, moving, and renaming of media is easier now and requires no special wiki function.
+However, this too has to undergo the pull request protocol.
+While osu!wiki will resize images to the width size limit if the image is too large, specific media size requirements must be handled manually.
+
+The most damaging change for pages that use it liberally would be **no Templates support**.
+You must manually duplicate the template and paste it accordingly with proper changes to the article you are editing, _on every language file_.
+
+Tables are supported by a bare-minimum with GFM (without extensions), unlike MediaWiki.
+
+Markdown is sensitive, as using the wrong capitalization in links can break them (i.e. links and images).
```

Comment:
L126: Rephrase opening phrase. ``Markdown is sensitive`` => ``Markdown is case-sensitive``

be a bit verbose.